tests/test_utils.py

```diff
@@ -1485,6 +1501,26 @@ def test_create_pddl():
 """
 
 
+def test_VideoMonitor():
+    """Tests for VideoMonitor()."""
+    env = CoverEnv()
```

Comment:
Videos are typically quite slow to generate. How slow is this test?

Reply:
0.77 seconds (note it's only 5 frames)

Reply:
I changed to 2 frames and it's now 0.43 seconds